packages/core/src/app/ovm/deciders/db-decider.ts

```diff
@@ -0,0 +1,69 @@
+import { Decider, Decision } from '../../../types/ovm/decider.interface'
+import { Bucket, DB } from '../../../types/db'
+import { Md5Hash } from '../../utils'
+
+export abstract class DbDecider implements Decider {
```

Comment:
We could consider calling this a `KVDecider` ? Because specifically it specializes in key value pairs